pkg/updatestatus/nodeinformer_test.go

```diff
@@ -149,6 +151,23 @@ func getMCP(name string) *machineconfigv1.MachineConfigPool {
 	}
 }
 
+func (c *nodeInformerController) initializeCaches() error {
```

Comment:
placing `nodeInformerController` method in `_test.go` file does not look right

Reply:
I think It is indeed a bug. I made a separate commit for its fix.

https://github.com/openshift/cluster-version-operator/pull/1164/commits/998d96cb2a0bb8f80fcf40df15673953fd1180c3